File: Backend/Financeiro.ConciliacaoWebAPI/JsonWebToken/Code/UnidadeConta/UnidadeContaBLL.cs
using JsonWebToken.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace JsonWebToken.Code.UnidadeConta
{
  public class UnidadeContaBLL
  {
    public List<UnidadeContaModel> SelectUnidadeConta()
    {
      try
      {
        UnidadeContaDAL dal = new UnidadeContaDAL();
        return dal.SelectUnidadeConta();
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
    public void AtivarUnidadeConta(UnidadeContaModel unidadeConta)
    {
      try
      {
        UnidadeContaDAL dal = new UnidadeContaDAL();
         dal.AtivarUnidadeConta( unidadeConta);
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
    public void InativarUnidadeConta(UnidadeContaModel unidadeConta)
    {
      try
      {
        UnidadeContaDAL dal = new UnidadeContaDAL();
         dal.InativarUnidadeConta( unidadeConta);
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
    public void InsertUnidadeConta(UnidadeContaModel unidadeConta)
    {
      try
      {
        UnidadeContaDAL dal = new UnidadeContaDAL();
         dal.InsertUnidadeConta(unidadeConta);
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
  }
}

File: Backend/Martins.Conciliacao.API/Code/UnidadeConta/UnidadeContaBLL.vb
Imports Martins.Conciliacao.Model

Namespace Code.UnidadeConta
  Public Class UnidadeContaBLL
    Public Function SelectUnidadeConta() As List(Of UnidadeContaModel)
      Try
        Dim dal As New UnidadeContaDAL()
        Return dal.SelectUnidadeConta()
      Catch ex As Exception
        Throw ex
      End Try
    End Function
    Public Sub AtivarUnidadeConta(unidadeConta As UnidadeContaModel)
      Try
        Dim dal As New UnidadeContaDAL()
        dal.AtivarUnidadeConta(unidadeConta)
      Catch ex As Exception
        Throw ex
      End Try
    End Sub
    Public Sub InativarUnidadeConta(unidadeConta As UnidadeContaModel)
      Try
        Dim dal As New UnidadeContaDAL()
        dal.InativarUnidadeConta(unidadeConta)
      Catch ex As Exception
        Throw ex
      End Try
    End Sub
    Public Sub InsertUnidadeConta(unidadeConta As UnidadeContaModel)
      Try
        Dim dal As New UnidadeContaDAL()
        dal.InsertUnidadeConta(unidadeConta)
      Catch ex As Exception
        Throw ex
      End Try
    End Sub
  End Class
End Namespace
